src/main/java/pl/edu/agh/mwo/reporter/loader/DataLoader.java

```diff
@@ -19,7 +19,8 @@
 
     public Company loadData(List<Path> paths, LocalDate dateFrom, LocalDate dateTo, String filterByEmployee) throws IOException {
         Company company = new Company();
-
+        System.out.println(">> Raport błędów w plikach wejściowych Excel <<");
```

Comment:
poprzednio usunełam to bo to sie wypisuje też jeśli nie ma żadnych błędów, więc chyba najprościej usunąć te 2 linie

Reply:
ok